lib/trino-hdfs/src/main/java/io/trino/hdfs/azure/HiveAzureConfig.java

```diff
@@ -33,29 +33,29 @@ public class HiveAzureConfig
     private String abfsOAuthClientId;
     private String abfsOAuthClientSecret;
 
-    @Deprecated
+    @Deprecated(forRemoval = true, since = "470")
```

Comment:
Oh .. didnt know about that.. what does it do?

Reply:
Also ... kinda weird btw.. forRemoval = false would never make sense to me

Reply:
Cool .. could we add a strict mode that causes a startup to fail if deprecated properties are used?

Reply:
we can

Reply:
I think that would be a nice feature to have .. and we could tell users to run it for their testing/deploymnent but have it off by default .. and we could even set it for our test runs